test/utils.js

```diff
@@ -63,7 +63,9 @@ async function startNext(appPath, port) {
  * @returns Promise<void>
  */
 async function cleanup(appPath) {
-  await fs.rm(path.resolve(appPath, './.next'), { recursive: true });
+  try {
+    await fs.rm(path.resolve(appPath, './.next'), { recursive: true });
+  } catch {}
```

Comment:
Which error is expected to be thrown here?

Reply:
If you run the tests the first time (no .next directory) it will throw `.next directory does not exist`

Reply:
then we need to catch it as such

```suggestion
  } catch (err) {
    if (err.code !== 'ENOENT') {
      throw err;
    }
  }
```